packages/calling-stateful-client/package.json

```diff
@@ -35,10 +35,10 @@
     "immer": "9.0.6"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.4.3-beta.1 || 1.3.2"
+    "@azure/communication-calling": "1.4.3-beta.1 || 1.4.3"
```

Comment:
I've asked this before but forget, can these be peer deps be `^1.4.3` so customers can use newer calling sdk versions without having to upgrade?

Reply:
Definitely not in this PR.

For stable dep, we can move to minimum-supported-sdk-version. The key decision there is about testing coverage and support expectations -- our CI can only test against some versions (in fact we'll want to test against the oldest supported version at any point) so we have to make sure we set expectations accordingly.

Reply:
I'd like to do this for stable deps. @alkwa-msft  thoughts?